server/controllers/userController.js

```diff
@@ -0,0 +1,57 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcryptjs';
+import config from '../../config';
+import users from '../models/users';
+import db from '../models/database';
+
+class userController {
+  static createUser(req, res) {
+    const {
+      username, email, phone, address, password,
+    } = req.body;
+
+    const hashedPassword = bcrypt.hashSync(password, 10);
+    db.query(`SELECT username from users WHERE username = '${username}'`)
+      .then((data) => {
+        if (data.rowCount > 0) {
+          res.status(409).json({
+            message: 'Username already exists. Use a different Username',
+          });
+        }
+      })
+      .catch(err => console.error(err.stack));
+    
+    db.query(`SELECT email from users WHERE email = '${email}'`)
```

Comment:
This block of code is similar to that on line 16. can't you put them in the same block?
_Hint_
 use the _OR_ operator.

Reply:
Thanks for the hint. I have refactored it.